CageMatch/CageMatchTests/NavigationTests.swift

```diff
@@ -92,13 +92,16 @@ class NavigationTests: XCTestCase {
             XCTFail("Could not get the BrokenMainNavigationItemJSON")
             return
         }
-
+        
         do {
             let _ = try NavigationResponseMarshal(object: json)
         }
         catch {
             XCTAssertNotNil(error, "Navigation response missing main navigation should fail")
```

Comment:
this wasn't actually catching any errors b/c a successful parse of the ```NavigationResponseMarshal``` wouldn't throw.